plugins/techdocs-module-addons-contrib/src/LigthBox/LightBox.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useEffect } from 'react';
+import { useShadowRootElements } from '@backstage/plugin-techdocs-react';
+// @ts-ignore
```

Comment:
There seems to be a [@types/photoswipe](https://www.npmjs.com/package/@types/photoswipe) - does it work to add that as a dependency and removing the ts-ignore line?

Reply:
Doesn't solve the issue and I think it's only for version 4. We are using 5.